src/marqo/s2_inference/model_registry.py

```diff
@@ -1734,6 +1735,15 @@ def _get_random_properties() -> Dict:
     }
     return RANDOM_MODEL_PROPERTIES
 
+
+def _get_no_model_properties() -> Dict:
```

Comment:
The reason is that when user just provided `no_model` without any `model_properties`, it will not tell the users that `this model does not exist` but say `dimensions is required`. Also, we record all the supported models in the `model_registry`.

Reply:
Got it, sounds good.